src/Bicep.Core/Emit/EmitterSettings.cs

```diff
@@ -22,6 +22,8 @@ public EmitterSettings(SemanticModel model)
                 model.Features.SymbolicNameCodegenEnabled ||
                 // there are any user-defined type declarations
                 model.Root.TypeDeclarations.Any() ||
+                // there are any user-defined function declarations
+                model.Root.FunctionDeclarations.Any() ||
```

Comment:
This isn't strictly necessary, but it's not a bad idea for increasing symbolic name adoption.

Reply:
Yeah that was the goal. Might as well!